### PR TITLE
make the query result clickable if certname is in it

### PR DESCRIPTION
**Pull request by GermanG**

### CODE REVIEW

**Comment by gdubicki** [comment]
Hi @GermanG , thank you for your contribution! Please check why the tests are failing though and a comment that I will post in a moment.

File: puppetboard/utils.py [hunk 1]
```diff
@@ -94,7 +94,10 @@ def prettyprint(value):
     for e in value:
         html += "<tr>"
         for k in e:
-            html += "<td>" + formatvalue(e[k]) + "</td>"
+            if k == "certname":
+                html += "<td> <a href = '/node/" + formatvalue(e[k]) + "' >" + formatvalue(e[k]) + "</a></td>"
```
Comment (gdubicki) [inline puppetboard/utils.py:98]: Please use `url_for()` Flask's function to generate the target URL. See https://flask.palletsprojects.com/en/1.1.x/quickstart/#url-building for more info on how and why. :)

Comment (GermanG) [reply puppetboard/utils.py]: Thanks for the pointer, I think the code style is also making the linter happy, but I'm really a newbie regarding unit testing, can you please point me in the right direction there?

**Comment by gdubicki** [comment]
Thanks for fixing all the issues and making the PR all green and shiny! 😊

**Comment by gdubicki** [comment]
Hey @GermanG , you can start using your new feature in [v3.4.0 RC1](https://github.com/voxpupuli/puppetboard/releases/tag/v3.4.0rc1) released a moment ago. ☺️

**Pull request merged**